Framework/Crystal/test/SaveIsawUBTest.h

```diff
@@ -147,4 +148,49 @@ class SaveIsawUBTest : public CxxTest::TestSuite {
 
     remove(File2.c_str());
   }
+
+  void test_exec_rotate_by_gonio() {
+    // Fake output WS
+    std::string wsname = "ws_rot_gonio";
+    MatrixWorkspace_sptr ws = WorkspaceCreationHelper::create2DWorkspace(10, 10);
+    AnalysisDataService::Instance().addOrReplace(wsname, ws);
+
```

Comment:
Is it worth testing in the case that the input workspace is a peak workspace? Up to you, I know the two cases are very similar.

Reply:
Yep, currently there is no test saving a UB from a peaks workspace at all so I will add one